app/views/admin/registrations/index.xlsx.axlsx

```diff
@@ -3,7 +3,7 @@ wb.add_worksheet(name: 'registrations') do |sheet|
   bold_style = wb.styles.add_style(b: true)
   row = ['Attended', 'Name', 'Nickname', 'Affilιation', 'Email', 'Arrival', 'Departure']
 
-  @conference.questions.each do |question|
+  @conference.questions.sort_by(&:id).each do |question|
```

Comment:
This is not related with the performance, right? I'd move it to a different commit

Reply:
In order to improve parity with the old design, the questions needed to be sorted consistently.